stubs/pyaudio/pyaudio.pyi

```diff
@@ -64,7 +67,45 @@ paOutputUnderflow: Final[int]
 paOutputOverflow: Final[int]
 paPrimingOutput: Final[int]
 
-paMacCoreStreamInfo: PaMacCoreStreamInfo
+paFramesPerBufferUnspecified: Final[int]
+
+if sys.platform == "darwin":
+    @final
+    class paMacCoreStreamInfo:
+        paMacCoreChangeDeviceParameters: Final[int]
+        paMacCoreFailIfConversionRequired: Final[int]
+        paMacCoreConversionQualityMin: Final[int]
+        paMacCoreConversionQualityMedium: Final[int]
+        paMacCoreConversionQualityLow: Final[int]
+        paMacCoreConversionQualityHigh: Final[int]
+        paMacCoreConversionQualityMax: Final[int]
+        paMacCorePlayNice: Final[int]
+        paMacCorePro: Final[int]
+        paMacCoreMinimizeCPUButPlayNice: Final[int]
+        paMacCoreMinimizeCPU: Final[int]
+        flags: Final[int]
+        channel_map: Final[_ChannelMap | None]
+        def __init__(self, flags: int | None = ..., channel_map: _ChannelMap | None = ...) -> None: ...
```

Comment:
```suggestion
        def __init__(self, flags: int = ..., channel_map: _ChannelMap = ...) -> None: ...
```
Tried it locally, passing None doesn't work.

Reply:
You're right. Since I'm moving these around, may as well fix them. I've also double-checked that `PaMacCoreStreamInfo.__init__` is correct.